layouts/partials/slideshow.html

```diff
@@ -1,14 +1,15 @@
+<!--slideshow, css found here rel="../../assets/css/extended/slideshow.css"-->

```

Comment:
Unnecessary comment that will be confusing in the future if the path changes

Reply:
request complete!